projects/angular/components/ui-suggest/src/ui-suggest.component.ts

```diff
@@ -1105,6 +1119,13 @@ export class UiSuggestComponent extends UiSuggestMatFormFieldDirective
         }
     }
 
+    computeDisplayValue() {
+        return this.value.length > 0
+            ? this.value.map(v => this.intl.translateLabel(v.text))
+                .join(',')
```

Comment:
could we extract and reuse from [here](https://github.com/UiPath/angular-components/blob/3fb40552de32cd71d6101e4d7aa0208af89b5740/projects/angular/components/ui-suggest/src/ui-suggest.component.ts#L312)? 
```suggestion
                .join(', ')
```

Reply:
done!